_templates/provider/new/package.ejs.t

```diff
@@ -15,7 +15,9 @@
   "license": "MIT",
   "keywords": [],
   "scripts": {
+    "prebuild": "rimraf build",
     "build": "run-p build:*",
+    "rebuild": "npm run prebuild && npm run build",
```

Comment:
We can remove this, because npm will call prebuild  by default when calling build